modin/core/storage_formats/pandas/utils.py

```diff
@@ -48,7 +54,7 @@ def compute_chunksize(df, num_splits, default_block_size=32, axis=None):
         DataFrame to split.
     num_splits : int
         Number of splits to separate the DataFrame into.
-    default_block_size : int, default: 32
+    default_block_size : int, optional
         Minimum number of rows/columns in a single split.
```

Comment:
```suggestion
        Minimum number of rows/columns in a single split.
        If not specified, the value is assumed equal to ``MinPartitionSize``.
```
or something along these lines to indicate where we fill the default from.